events.rst

```diff
@@ -9,3 +9,51 @@ Exhibition days happen occasionally. On these days we ask members to come into t
 There are regular workshops, talks and groups on various subjects, which are often member-led and typically open to the public as well. Upcoming events are promoted on our social media, in the newsletter, the Google Group, and on the Nottinghack homepage. We'd love to hear from you if you're interested in running a workshop, a talk, or a regular group.
 
 Aiming to run an event? Contact the Trustees (a requirement for a public facing event) and read about the process for events on the wiki which includes guidelines for risk assessments and how we advertise and ticket events: https://wiki.nottinghack.org.uk/wiki/Events
+
+Typical Events and Themes
+-------------------------
+
+The best way to stay up to date with events in the hackspace is to join our `Discord <online.html#discord-https-wiki-nottinghack-org-uk-discord>`_ server. We try to post these on our website too on the `What's On <https://nottinghack.org.uk/calender/>`_ page.
+
+Member's Meeting
+^^^^^^^^^^^^^^^^
+
+All members are invited to the monthly members meeting. These occur on the first Wednesday of each month at 20:00. You can attend remotely via Zoom, or just us in the Blue Room downstairs. The agenda is published at least a few days prior to the meeting, which can be found on the Wiki `Agenda <https://wiki.nottinghack.org.uk/index.php?title=Agenda>`_ page.
```

Comment:
Existing content, but we got the apostrophe right in the title, then wrong in the first sentence. 😅